app/frontend/styles/_primary-navigation.scss

```diff
@@ -88,6 +99,7 @@
 
       &:before {
         background-color: govuk-colour("black");
+        height: 7px;
```

Comment:
Shouldn't we be using `rem` instead of pixels so font size is scaled correctly across browsers?

Reply:
I think it's fine - we use pixels in many places that do not impact the font size such as margin, padding and borders.

Reply:
The design system uses `px` for most units - I know for a good reason, but I don't remember them any more. I think all modern browsers will scale them fine.